projects/example-app/src/app/books/index.ts

```diff
@@ -1 +0,0 @@
-export * from './books.module';
```

Comment:
I had to remove these index files because they weren't imported. 
The alternative would be to add these files to the config.